docs/cloud-native-security/kspm-findings.asciidoc

```diff
@@ -21,10 +20,12 @@ By default, the Findings page lists all findings, without grouping or filtering.
 [discrete]
 === Group findings
 
-. Click **Group findings by:** to open a list of fields by which you can group findings. Select one of the suggested group-by fields, or click **Custom field** to choose your own group-by field. 
-. When grouping is turned on, click a group to expand it and examine all findings within that group.
+. Click **Group vulnerabilities by:** to open a list of fields by which you can group your data. Select one of the suggested fields, or click **Custom field** to choose your own. You can select up to three group-by fields at once. 
+. When grouping is turned on, click a group to expand it and examine all sub-groups or findings within that group.
 . To turn off grouping, click **Group findings by:** and select **None**.
 
+NOTE: Multiple groupings apply to your data in the order you selected them; for example if you first select **Cloud account**, and then select **Resource**, the top-level grouping will be based on **Cloud account**, and its subordinate grouping will be based on **Resource**. 
```

Comment:
for a better example closer to the real scenario I would suggest you replace **Cloud account** with **Kubernetes cluster** (for the KSPM docs only):


NOTE: Multiple groupings apply to your data in the order you selected them; for example if you first select **Kubernetes cluster**, and then select **Resource**, the top-level grouping will be based on **Kubernetes cluster**, and its subordinate grouping will be based on **Resource**.